service/src/main/java/uk/gov/hmcts/reform/fpl/service/MigrateCaseService.java

```diff
@@ -424,4 +489,44 @@ private String stripIllegalCharacters(String str) {
         return str.replace("<", "")
             .replace(">", "");
     }
+
+    public Map<String, Object> migrateHearingType(CaseData caseData) {
+        if (nonNull(caseData.getHearingDetails())) {
+            List<Element<HearingBooking>> hearingDetails = caseData.getHearingDetails();
+            for (Element<HearingBooking> hearings: hearingDetails) {
+                HearingBooking hearingBooking = hearings.getValue();
+                if (OTHER.equals(hearingBooking.getType())) {
+                    Optional<HearingType> hearingType = evaluateType(hearingBooking.getTypeDetails());
```

Comment:
I think if we just did a get on the map it'll either return a value or null automatically, without you having to explicitly set it to null